### PR TITLE
Add internal CloseButton and migrate components on it

### PR DESCRIPTION
**Pull request by Faradey27**
### What changed

...

### Why it changed

...

---

- [ ] Change is tested
- [ ] Change is documented
- [ ] Build is green
- [ ] Change of UX is approved by [wuwa](https://github.com/wuwa) or [milkyfruit](https://github.com/milkyfruit)

### CODE REVIEW

**Comment by Faradey27** [comment]
@shlomitc as we discussed I have created an internal CloseButton component till we will start to clean up buttons.

**Comment by shlomitc** [comment]
cool, that's great. just the very minimal refactor that was needed at the moment.

I think there is an issue with the shadow of the marketerial modal
image

**Comment by Faradey27** [comment]
@shlomitc fixed

File: src/MessageBox/MessageBoxMarketerialLayout.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import WixComponent from '../BaseComponents/WixComponent';
 import classNames from 'classnames';
-import CloseLarge from 'wix-ui-icons-common/system/CloseLarge';
+import CloseButton from './../CloseButton';
```
Comment (shlomitc) [inline src/MessageBox/MessageBoxMarketerialLayout.js]: don't need the `./` in the start. use `../` directly

File: src/Notification/Notification.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ import classNames from 'classnames';
 
 import WixComponent from '../BaseComponents/WixComponent';
 import {children, once, optional} from '../../src/Composite';
-import CloseButton from './CloseButton';
+import CloseButton from './../CloseButton';
```
Comment (shlomitc) [inline src/Notification/Notification.js]: also here

**Pull request merged**